### ISSUE TITLE
Segmentation fault on single file

### ISSUE THREAD

**Issue by dhoconno** [PacificBiosciences/pbAA#4]
Hi there, I recently processed ~185 samples with pbaa. Since nearly all of the samples processed successfully I think the syntax is generally OK. One sample consistently caused a segmentation fault with no additional explanation.

I used pbaa-0.1.2-0 with the command:

```
pbaa cluster \
24923-vsearch-07-mhc-I.fasta \
MD130.fastq \
MD130
```

A copy of the query and guide file (plus their indices) can be downloaded from: https://drive.google.com/file/d/1WcC7hgxTo6m-U2jz-YH4pQGGLaZ-4DFG/view?usp=sharing

It isn't critical that I get data from this sample, except that having one sample fault unexpectedly makes my overall workflow more fragile than I'd like.

Thanks!

--Dave

**Comment by zeeev** [comment]
Hi @dhoconno,

Do you know if version 1.0.0 fixes this issue?

**Comment by zeeev** [comment]
Hi @dhoconno ,

Good news the issue was resolved in v1.0.0.

```
>|> 20210428 20:46:08.295 -|- INFO -|- LaaRunner -|- 0x117b60e00|| -|- Finished without error.
>|> 20210428 20:46:08.295 -|- INFO -|- LaaRunner -|- 0x117b60e00|| -|- Real time     : 12m 55s
>|> 20210428 20:46:08.295 -|- INFO -|- LaaRunner -|- 0x117b60e00|| -|- CPU time      : 59m 32s
>|> 20210428 20:46:08.295 -|- INFO -|- LaaRunner -|- 0x117b60e00|| -|- Peak RSS      : 0.847 GB
```

I did notice a number of warnings while running the code. A good number of reads were not placed. Perhaps there several alleles in the guides that are too close in sequence identity, or maybe a guide is contained within another, voiding unique kmers.
